dotnet6/sns/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -16,6 +16,9 @@ Transform:
 Globals:
   Function:
     Timeout: 100
+  Api:
```

Comment:
I don't think we need to add it in this template since this template doesn't contain any Api Event.